includes/blocks/class-sensei-block-take-course.php

```diff
@@ -29,12 +29,22 @@ public function __construct() {
 	 * @access private
 	 */
 	public function register_block() {
-		register_block_type(
-			'sensei-lms/button-take-course',
-			[
-				'render_callback' => [ $this, 'render_take_course_block' ],
-			]
-		);
+
+		/**
+		 * Filter the render callback of the Take Course block.
+		 *
+		 * In WordPress versions 5.5 and later, block type arguments can be filtered by using register_block_type_args
+		 * instead. This filter exists to support earlier versions only.
+		 *
+		 * @since 3.6.0
+		 * @hook sensei_take_course_block_type_args
+		 * @see register_block_type
+		 *
+		 * @param array $args The block arguments as defined by register_block_type.
```

Comment:
I think it should be `{array}` instead, because of the hooks documentation.

Reply:
Fixed in d05bb2c649d8d925b2f83e63d1cee36c7f76e4e7